core/unit_test/CMakeLists.txt

```diff
@@ -782,9 +782,15 @@ if(Kokkos_ENABLE_CUDA)
 
   kokkos_add_executable_and_test(
     CoreUnitTest_CudaTimingBased SOURCES UnitTestMainInit.cpp cuda/TestCuda_DebugSerialExecution.cpp
-    cuda/TestCuda_DebugPinUVMSpace.cpp
   )
 
+  # Do not run this test on Jetson devices since this architecture has shared memory
+  if(NOT KOKKOS_ARCH_AMPERE87)
+      kokkos_add_executable_and_test(
```

Comment:
We generally prefer to still compile the test and rather add something like
```
#ifdef  KOKKOS_ARCH_AMPERE87
  GTEST_SKIP() << "skipping for Jetson devices that have integrated memory ".
#endif
```
to the test body.